src/js/components/reviewData/ReviewDataNotifyModal.jsx

```diff
@@ -38,6 +38,9 @@ export default class ReviewDataNotifyModal extends React.Component {
             users: null,
             selectedUsers: []
         };
+
+        this.generateMailToLink = this.generateMailToLink.bind(this);
```

Comment:
We don't need a binding for this function since it's only used within the component.